src/MCPClient/lib/clientScripts/restructureForCompliance.py

```diff
@@ -21,34 +21,113 @@
 # @author Joseph Perry <joseph@artefactual.com>
 
 import os
+import re
 import sys
 import shutil
 
+import django
+django.setup()
+from main.models import Transfer
+
+from verifyBAG import verifyBAG
```

Comment:
Looks like this one also didn't rebase cleanly on #393 since I also did some bag cleanup there.